lib/webpack_manifest/rails/configuration.rb

```diff
@@ -87,12 +109,44 @@ def manifests
         repo
       end
 
+      # Resolve base_path as an absolute path
+      #
+      # @return [String]
+      def resolved_base_path
+        File.expand_path(base_path || '.', root_path)
+      end
+
+      # Resolve watched_paths as absolute paths
+      #
+      # @return [Array<String>]
+      def resolved_watched_paths
+        base = resolved_base_path
+        watched_paths.map { |path| File.expand_path(path, base) }
+      end
+
+      # @return [String]
+      def cache_path
+        root_path.join('tmp', 'cache', 'webpack_manifest').to_s
```

Comment:
`root_path` should be a String, same as the other path params, instead of Pathname.